server/tasks/github_webhook_tasks.py

```
@@ -0,0 +1,934 @@
+    handling boilerplate at every call site. A missing path returns
+    ``None``, which ``_fmt_field`` then renders as ``<missing>``.
+    """
+    current: Any = payload
+    for key in keys:
+        if not isinstance(current, dict):
+            return None
+        current = current.get(key)
+    return current
+
+
+def _fmt_field(value: Any) -> str:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)